locales/en/homepage.json

```
@@ -36,6 +36,9 @@
   "reuses.tosdr.author": "ToS;DR",
   "reuses.tosdr.subtitle": "Lets users collaboratively rate how respectful terms of services are of their rights.",
   "reuses.tosdr.title": "Terms of Service; Didn’t Read",
+  "reuses.report-lab-lex.author": "Lab-LEX",
+  "reuses.report-lab-lex.title": "Governance and regulation of social media",
+  "reuses.report-lab-lex.subtitle": "Exploratory study of the evolution of social media terms of use and their impact on human rights.",
```

That research paper is an exploratory study, but I believe from the visitor's perspective  it's important to insist on the academic aspect of this work 🙂 
```suggestion
  "reuses.report-lab-lex.subtitle": "Research paper on the impact on human rights of the evolution of social media terms of use.",
```